src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -312,6 +313,22 @@ protobuf_schema_definition::raw() const {
     return canonical_schema_definition::raw_string{_impl->fd->DebugString()};
 }
 
+::result<ss::sstring, kafka::error_code>
+protobuf_schema_definition::name(std::vector<int> const& fields) const {
+    if (fields.empty()) {
+        return kafka::error_code::invalid_record;
+    }
+    auto f = fields.begin();
+    auto d = _impl->fd->message_type(*f++);
+    while (fields.end() != f && d) {
+        d = d->nested_type(*f++);
+    }
+    if (!d) {
+        return kafka::error_code::invalid_record;
+    }
+    return d->full_name();
+}
+
```

Comment:
It's not possible for a customer using the REST-ish API, and it's not possible to be represented on the topic, since the schema is inlined into the subject-version representation.